### PR TITLE
Add tests covering local container networking.

### PR DESCRIPTION
**Pull request by robshakir**
Adds a new testplan and container networking tests demonstrating being able to dial local services from container running on a network device.

### CODE REVIEW

File: feature/container/containerz/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## CNTR-1.1: Deploy and Start a Container
+
+Using the
+[`gnoi.Containerz`](https://github.com/openconfig/gnoi/tree/main/containerz) API
+(reference implementation to be available
+[`openconfig/containerz`](https://github.com/openconfig/containerz), deploy a
+container to the DUT. Using `gnoi.Containerz` start the container.
+
+The container should expose a simple health API. The test succeeds if is
+possible to contain the container via the gRPC API to determine its health.
```
Comment (alshabib) [inline feature/container/containerz/README.md:12]: connect to?

**Review by alshabib — APPROVED** [review]

**Comment by alshabib** [comment]
Thank you for doing this!

File: feature/container/containerz/README.md (hunk 1 above)
Comment (robshakir) [reply feature/container/containerz/README.md:12]: Fixed!

**Comment by robshakir** [comment]
@alshabib PTAL again, thx!

**Comment by robshakir** [comment]
Linter is running against `pb.go` files, so there will be static analysis errors here.

**Pull request merged**